include/rocksdb/table.h

```diff
@@ -547,6 +547,29 @@ struct BlockBasedTableOptions {
 
   PrepopulateBlockCache prepopulate_block_cache =
       PrepopulateBlockCache::kDisable;
+
+  // RocksDB does auto-readahead for iterators on noticing more than two reads
+  // for a table file if user doesn't provide readahead_size. The readahead
+  // starts at 8KB and doubles on every additional read upto
+  // max_auto_readahead_size and max_auto_readahead_size can also be configured.
+  //
+  // Special Value: 0 - If initial_auto_readahead_size is set 0 then no implicit
```

Comment:
This might be a bit confusing. What if this is set to 0, but max_auto_readahead_size is non-zero? Is it possible to sanitize the values?

Reply:
In `BlockPrefetcher`,  If `initial_auto_readahead` is set 0 then it doesn't prefetch  and returns from there, irrespective of `max_auto_readahead_size`. What values are you suggesting in the sanitization?

Reply:
I mean, if initial_auto_readahead_size is 0, check that max_auto_readahead_size is also 0, and vice versa. Maybe just mention in the comment that either one set to 0 will disable prefetching.

Reply:
Sure. I will update the comments to explicitly mention it will disable the internal prefetching.